apps/discord/win.py

```diff
@@ -7,6 +7,13 @@
 app: discord
 """
 
+ctx.lists["user.discord_destination"] = {
```

Comment:
Should these be available for macOS too?

Reply:
yeah that's true, for some reason I tried to do this before and it didn't work, but I got it to work this time